x-pack/docs/en/security/securing-communications/security-basic-setup.asciidoc

```diff
@@ -0,0 +1,277 @@
++
+Each node directory contains a single `<node-name>.p12` certificate file, which contains the node certificate, instance private key, and CA certificate.
+
+. Copy the appropriate `<node-name>.p12` certificate file to the
+   `ES_PATH_CONF` directory on each node.
++
+WARNING: This file must reside in the `ES_PATH_CONF` directory or {es}
+will not start.
+
+*Next*: <<encrypt-internode-communication>>
+
+[[encrypt-internode-communication]]
+==== Encrypt internode communications with TLS
+
+The transport networking layer is used for internal communication between
+nodes in a cluster. When security features are enabled, you must use TLS to
+ensure that communication between the nodes is encrypted.
+
+Now that you've generated a certificate authority and certificates, you'll
+update your cluster to use these files.
+
+NOTE: Complete the following steps for each node in your cluster. To join the
+same cluster, all nodes must share the same `cluster.name` value.
+
+.  Open the `ES_PATH_CONF/elasticsearch.yml` file and make the following
+changes:
+
+   a. Add the `cluster-name` setting and enter a name for your cluster:
++
+[source,yaml]
+----
+cluster.name: my-cluster
+----
+
+b. Add the `node.name` setting and enter the name of the certificate that
+you generated for this node. For simplicity, it's good practice for this value
+to match the certificate name that you defined in your `certificates.yaml` file:
++
+[source,yaml]
+----
+node.name: node-1
+----
+
+c. Add the following settings to enable internode communication and provide
+access to the node's certificate:
++
+[source,yaml]
+----
+xpack.security.transport.ssl.enabled: true
+xpack.security.transport.ssl.verification_mode: certificate
+xpack.security.transport.ssl.client_authentication: required
+xpack.security.transport.ssl.keystore.path: <node-name>.p12
+xpack.security.transport.ssl.truststore.path: <node-name>.p12
+----
+
+.  If you entered a password when creating the node certificate, run the following commands to store the password in the {es} keystore:
++
+--
+[source,shell]
+----
+./bin/elasticsearch-keystore add xpack.security.transport.ssl.keystore.secure_password
+----
+
+[source,shell]
+----
+./bin/elasticsearch-keystore add xpack.security.transport.ssl.truststore.secure_password
+----
+--
+
+.  Complete the previous steps for each node in your cluster.
+
+.  Restart {es}. The method for <<starting-elasticsearch,starting>> and <<starting-elasticsearch,stopping>> {es} varies depending on how you installed it.
++
+For example, if you installed {es} with an archive distribution
+(`tar.gz` or `.zip`), you can enter `Ctrl+C` on the command line to stop
+{es}.
++
+WARNING: You must perform a full cluster restart. Nodes that are configured to
+use TLS for transport cannot communicate with nodes that use unencrypted transport connection (and vice-versa).
+
+*Next*: <<add-built-in-users,Add the built-in users to {kib}>>
+
+[[add-built-in-users]]
+==== Add the built-in users to {kib}
```

Comment:
I don't love this title for 2 reasons

1. We're really only adding 1 user (not "users")
2. Some people get confused about whether Kibana users and ES users are the same thing (they are), so a heading that says you need to "add .. users to Kibana" can give the wrong impression.

My best suggestion is this, maybe someone has a better idea.
```suggestion
==== Configure {kib} to connect to {es} with a password
```